src/System.Private.Xml/src/System/Xml/Schema/XmlSchemaAnyAttribute.cs

```diff
@@ -17,7 +17,7 @@ public class XmlSchemaAnyAttribute : XmlSchemaAnnotated
         [XmlAttribute("namespace")]
         public string Namespace
         {
-            get { return _ns ?? NamespaceList.ToString(); }
+            get { return _ns ?? NamespaceList?.ToString(); }
```

Comment:
was this returning null or empty string before?

Reply:
It was returning null.